lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_hosts/recipes/default.rb

```diff
@@ -16,23 +16,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+START = 60
```

Comment:
comment explaining what this is would be helpful.  potentially link to https://github.com/customink-webops/hostsfile/blob/master/libraries/entry.rb#L168